BUG: cibuildwheel fails to build wheels for Windows

**Describe the bug**
Failing to build wheels for Windows

**To Reproduce**
See latest action run for wheel building: https://github.com/QuakeMigrate/QuakeMigrate/actions/runs/7943189801/job/21687265047

```
Run pypa/cibuildwheel@v2.12.0
Run actions/setup-python@v4
Installed versions
Run pipx run --python "C:\hostedtoolcache\windows\Python\3.11.8\x64\python.exe" --spec "D:\a\_actions\pypa\cibuildwheel\v2.12.0" cibuildwheel "." --output-dir '"wheelhouse"' --config-file '""' --only '""'
creating virtual environment...
determining package name from 'D:\\a\\_actions\\pypa\\cibuildwheel\\v2.12.0'...
creating virtual environment...
installing cibuildwheel from spec 'D:\\a\\_actions\\pypa\\cibuildwheel\\v2.12.0'...
Invalid --only='""', must be a build selector with a known platform
Error: Process completed with exit code 1.
```

**Expected behavior**
Wheels should build without errors.

**Desktop (please complete the following information):**
 - Operating System: n/a
 - Python version: n/a
 - QuakeMigrate version: (1.0.2)

**Additional context**
This is a known bug with cibuildwheels -- https://github.com/pypa/cibuildwheel/issues/1740

Recommended fix is to update to latest version:

```diff
---        uses: pypa/cibuildwheel@v2.14.0
+++        uses: pypa/cibuildwheel@v2.16.5
```

--- a/.github/workflows/build_wheels.yml
+++ b/.github/workflows/build_wheels.yml
@@ -35,7 +35,7 @@ jobs:
           fetch-depth: 0
 
       - name: Build and test wheels
-        uses: pypa/cibuildwheel@v2.12.0
+        uses: pypa/cibuildwheel@v2.16.5
 
       - uses: actions/upload-artifact@v3
         with:

